pallets/automation-time/src/tests.rs

```diff
@@ -28,6 +28,11 @@ use sp_runtime::{
 	AccountId32,
 };
 
+type ActionOf<T> = Action<AccountId, Balance, <T as Config>::CurrencyId>;
+type TaskOf<T> =
+	Task<AccountId, Balance, <T as Config>::CurrencyId, <T as Config>::MaxExecutionTimes>;
+type MissedTaskV2Of = MissedTaskV2<AccountId, sp_core::H256>;
```

Comment:
```suggestion
use crate::{ActionOf, MissedTaskV2Of, TaskOf};
```
Need to update uses of MissedTaskV2Of to specify the generic arg for this to work `MissedTaskV2Of::<Test>`